### PR TITLE
Animation hem

### PR DESCRIPTION
**Pull request by CyrilleKolle**
Link to ticket: https://fixa.itiden.se/project/1362/tickets/85
Made another attempt at getting reverse animation.
Need tips on how to extend span element to get an extra margin out of the main div

### CODE REVIEW

**Review by ricllrstn — CHANGES_REQUESTED** [review]
We need to change the cubic-bezier values but we will need to experiment together, I think cubic-bezier(0.43, 0.83, 0.58, 1) can be a good start. Regarding the size of the highlight: can we adjust it to be the equivalent of when the text is selected with the mouse. Now the highlight can overlap and cause some letters (ie g) to be cut off. I like the space you added in the beginning and end of the highlight but can we cut that space in half. Is that space also possible to add to extremes of lines placed in the middle of a highlight if it for example takes up three rows? (If this last point is very complicated it might be worth to see it as a "nice-to-have"-feature and save it for later)

**Review by ricllrstn — CHANGES_REQUESTED** [review]
Nice. Can we try change the padding för the highligt to 6px for desktop and 3px för mobile? (6,6,6,6 & 3,3,3,3). Before next update lets adjust the cubic-bezier curve together, let me know when!

File: tsconfig.json [hunk 1]
```diff
@@ -36,5 +36,5 @@
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx"
-  ]
+, "components/Animation/BackgroundAnimationtsx"  ]
```
Comment (reekris) [inline tsconfig.json:39]: What is this? I think it should be removed

**Review by reekris — CHANGES_REQUESTED** [review]
Only ne minor change

**Review by ricllrstn — DISMISSED** [review]
Looking good! FYI: This branch should not go live until later.

**Review by reekris — APPROVED** [review]

**Pull request merged**